pandas/core/dtypes/common.py

```diff
@@ -68,9 +68,6 @@
 ensure_float64 = algos.ensure_float64
 ensure_float32 = algos.ensure_float32
 
-_ensure_datetime64ns = conversion.ensure_datetime64ns
-_ensure_timedelta64ns = conversion.ensure_timedelta64ns
```

Comment:
unrelated edit before i figured out what this branch was about; these are never used